brightchain-lib/src/lib/blocks/random.ts

```
@@ -14,12 +15,22 @@ import { RawDataBlock } from './rawData';
  * where data blocks are XORed with random blocks to make them meaningless on their own.
  */
 export class RandomBlock extends RawDataBlock {
-  private constructor(
+  protected static override checksumService: ChecksumService;
+
+  protected static override initialize() {
+    super.initialize();
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)